packages/swap/src/quoting/__tests__/Quoter.test.ts

```diff
@@ -286,14 +288,6 @@ describe(Quoter, () => {
       expect(mm.requestCount).toBe(0);
     });
 
-    it("respects the market makers's quoted assets", async () => {
```

Comment:
not sure if this was remove intentionally

Reply:
was duplicate